.github/workflows/vale.yml

```
@@ -0,0 +1,18 @@
+name: reviewdog
+on: push
```

```suggestion
on:
  pull_request:
  push:
```

I'm not sure without `pull_request` that it trigger a test whenever you create the PR for the first time

---

You're right, thanks! 🙏